8.0.Dockerfile

```diff
@@ -82,6 +86,7 @@ RUN pip install \
         virtualenv \
         wdb \
         simplejson \
+        geoip2 \
```

Comment:
nope, i did say it is not working for v8 (because GeoIP is no longer working with current data, that is only available in geoip2 format)

the v8 dockerfile is used for v7, v8, v9, v10 so to get it working in v9 and v10 i need to add the dependency into the v8 file:
```
lrwxrwxrwx  1 bigbear3001 bigbear3001    14 Mai 14  2018 10.0.Dockerfile -> 8.0.Dockerfile
-rw-rw-r--  1 bigbear3001 bigbear3001  8109 Feb 19 13:23 11.0.Dockerfile
-rw-rw-r--  1 bigbear3001 bigbear3001  7775 Feb 19 14:49 12.0.Dockerfile
-rw-rw-r--  1 bigbear3001 bigbear3001  7643 Feb 19 15:18 13.0.Dockerfile
lrwxrwxrwx  1 bigbear3001 bigbear3001    14 Jän 13 10:56 7.0.Dockerfile -> 8.0.Dockerfile
-rw-rw-r--  1 bigbear3001 bigbear3001  8398 Feb 19 13:25 8.0.Dockerfile
lrwxrwxrwx  1 bigbear3001 bigbear3001    14 Mai 14  2018 9.0.Dockerfile -> 8.0.Dockerfile
```